deploy/roles/pem-client/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
   when: install_pem
 
 - name: Ensure umask is correct
-  sudo: no
+  #sudo: no
```

Comment:
This was here to make sure that the `deploy` users was running the umask command as itself, and not as root.

Reply:
I'll revert it and test that, then.

Reply:
Reinstated (see above reply for more detail).